packages/textkit/src/utils/isUrl.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Checks if string is a URL
+ *
+ * @param  {string}  value value
+ * @return {Boolean} is URL
+ */
+const isUrl = value => {
+  // eslint-disable-next-line no-useless-escape
+  const regx = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g;
```

Comment:
Would be good to check the safety of this regexp since it's going to get applied on every word